com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/ParameterMetadata.java

```diff
@@ -23,4 +24,7 @@ default boolean getRequired() {
 
     ParameterType getType();
 
+    @Nullable
+    ParameterConverter getCustomConverter();
```

Comment:
Why custom? Isn't that the default converter for this parameter or?

Reply:
Nah, the converter that matches the type of the parameter is picked by the `ParameterConversion` class, if this one is `null`.